src/averageAges.js

```diff
@@ -37,7 +40,14 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter(({ sex, name }) =>
+      sex === 'f' && people.some(({ mother }) => name === mother))
+    : people.filter(({ sex }) => sex === 'f');
+
+  return women.reduce(
+    (prev, { born, died }) => prev + (died - born), 0)
```

Comment:
> I recommend using accumulator, acc or total for a parameter of reduce
[MDN Doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce)

The same one